scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -80,108 +79,112 @@
 y.value_counts(True)
```

Comment:
Are you seeing any of my other comments? or is this the only one that got saved?